src/snowcli/cli/loggers.py

```diff
@@ -0,0 +1,24 @@
+import logging
+
+
+DEFAULT_LOG_FORMAT = "%(asctime)s %(levelname)s %(message)s"
+DEBUG_LOG_FORMAT = "%(asctime)s %(levelname)s [%(name)s] %(message)s"
+
+
+def create(verbose: bool, debug: bool):
```

Comment:
```suggestion
def create_loggers(verbose: bool, debug: bool):
```
To be more explicite / verbose. Also can you please add short docstring describing what we do here?

Reply:
I thought it will be obvious if it is function in `loggers` module. Use of this will looks like:

```
loggers.create_loggers(...)
```

I will change and add